### ISSUE TITLE
Chap. 6.3, recurrent dropout, cannot reproduce book's results with TF2.2

### ISSUE THREAD

**Issue by CFGrote** [fchollet/deep-learning-with-python-notebooks#151]
Keras: 2.4.3
Tensorflow: 2.2

Running parts 6.3.6 (recurrent dropout) and 6.3.7 (stacked recurrent layers) of the [original notebook](https://github.com/fchollet/deep-learning-with-python-notebooks/blob/master/6.3-advanced-usage-of-recurrent-neural-networks.ipynb) gives the following training and validation losses:

6.3.6:
![image](https://user-images.githubusercontent.com/15476807/94893821-58fd7100-0488-11eb-9ef5-7260200750b8.png)

6.3.7:
![image](https://user-images.githubusercontent.com/15476807/94893850-687cba00-0488-11eb-91f2-43d5748e8e41.png)

How do we have to modify the hyperparameters to reproduce the book's results?

Attached is my environment.yml if someone wants to reproduce my findings.

[environment.yml.txt](https://github.com/fchollet/deep-learning-with-python-notebooks/files/5316291/environment.yml.txt)

**Comment by fangxuehouwuming** [comment]
Hi, I met this problem too. Have you solved it ?
